### DISCUSSION TITLE
Running docker compose using Docker-In-Docker, best practice

### DISCUSSION THREAD

**Discussion by RollsChris** [devcontainers/community#165] [Q&A]
Hi,

I'm trying to open a project within a devcontainer using docker-compose.yml.

The project also contains a docker-compose.yml i want to up and down.

First i mount a license from my host into the devcontainer ( using the devcontainers docker-compose.yml), like so:

```yml
volumes:
      - ../:/workspaces/flowable-kickoff:cached
      - ../license:/home/vscode/license
```

then within the other docker-compose.yml used in the project i have:

```yml
volumes:
      - ~/license:/home/node/license:ro
```

But when i do this license it becomes a directory and not a file and everything fails.

What am i doing wrong.

Basically i'm trying to mount something into my devcontainer and then from there into another container..

Can anybody help

Thanks.

**Comment by RollsChris** [comment]
Any ideas anyone, its really starting to annoy me. It docker-outside-docker has to be used with host paths?
